chromium_src/content/browser/private_aggregation/private_aggregation_manager.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CONTENT_BROWSER_PRIVATE_AGGREGATION_PRIVATE_AGGREGATION_MANAGER_H_
+#define BRAVE_CHROMIUM_SRC_CONTENT_BROWSER_PRIVATE_AGGREGATION_PRIVATE_AGGREGATION_MANAGER_H_
+
+#define GetManager                                    \
```

Comment:
see above

Reply:
Fixed in 68ce1fb3a03f3cb1fa9eb909384d547ac04b0d23